rabbitmq-server-3.13.yaml

```diff
@@ -21,17 +21,20 @@ environment:
       - build-base
       - busybox
       - ca-certificates-bundle
-      - elixir
+      - elixir-1.15
       - erlang-26
       - erlang-26-dev
       - libxslt
       - python3
       - rsync
+  environment:
+    # suppresses this message: "the VM is running with native name encoding of latin1 which may cause Elixir to malfunction as it expects utf8. Please ensure your locale is set to UTF-8 (which can be verified by running "locale" in your shell) or set the ELIXIR_ERL_OPTIONS="+fnu" environment variable"
+    ELIXIR_ERL_OPTIONS: "+fnu"
```

Comment:
que?! we should double check and ensure that we do in fact run in C.UTF-8 locale by default (glibc built-in / universally available). Will see what is missing and will add that. I thought we normally ship that in base-files, but possibly only present in the chainguard-images rather than in melange build containers.

Reply:
tried to fix this =/ this is really hard and confusing, took me longer than 60s to find the answer.